File: Programs/Python/calculator.py
first = float(input("Enter first numebr: "))
second = float(input("Enter second number: "))
operation = str(input("Enter operation(+,-,*,/): "))

if operation == "+":
	print(f"Result: {first + second}")

elif operation == "-":
	print(f"Result: {first - second}")

elif operation == "*":
	print(f"Result: {first * second}")

elif operation == "/":
	print(f"Result: {first / second}")

else:
	print("Unknown operation!")

File: Programs/C++/calculator.cpp
#include <iostream> 
using namespace std; 

int main() {
	float first;
	float second;
	string operation;
	
	cout << "Enter first number: ";
	cin >> first;
	cout << "Enter second number: ";
	cin >> second;
	cout << "Enter operation(+,-,*,/): ";
	cin >> operation;

	if (operation == "+") {
		cout << "Result: " << first + second;
	}

	else if (operation == "-") {
		cout << "Result: " << first - second;
	}

	else if (operation == "*") {
		cout << "Result: " << first * second;
	}

	else if (operation == "/") {
		cout << "Result: " << first / second;
	}

	else {
		cout << "Unknown operation!";
	}
}
